.github/workflows/canary.yml

```
@@ -72,3 +72,22 @@ jobs:
         with:
           name: test-results-${{ matrix.os }}-${{ matrix.suite }}-${{ matrix.shard }}-${{ matrix.configs }}
           path: /tmp/artifacts/*.json
+
+  report-results:
+    name: 'Report results'
+    needs: [canary-chrome-tests]
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    if: ${{ always() }}
```

Let's only post comment on failed runs.
```suggestion
    if: ${{ failure() }}
```